tee-worker/litentry/core/data-providers/src/geniidata.rs

```diff
@@ -82,9 +79,15 @@ impl GeniidataClient {
 	pub fn new() -> Self {
 		let mut headers = Headers::new();
 		headers.insert(ACCEPT.as_str(), "application/json");
-		headers.insert("api-key", GLOBAL_DATA_PROVIDER_CONFIG.read().unwrap().geniidata_api_key.as_str());
+		headers.insert(
+			"api-key",
+			GLOBAL_DATA_PROVIDER_CONFIG.read().unwrap().geniidata_api_key.as_str(),
+		);
```

Comment:
[off-topic] do you have an automatic syntax formatter running? how do I set up mine? 😄

Reply:
There's github hook `.githooks/pre-commit`